contracts/HundredVesting.sol

```diff
@@ -23,55 +23,50 @@ contract HundredVesting is Ownable {
         totalPeriod = totalPeriod / period;
     }
 
-    function getRemainedAmount (address vester) internal view returns (uint256) {
-        uint256 timestamp = addresses[vester].timestamp;
-        uint256 amount = (block.timestamp - timestamp) / timePerPeriod * addresses[vester].amount / totalPeriod;
-        return amount - addresses[vester].claimedAmount;
+    function getRemainedAmount (address vestedAddress) internal view returns (uint256) {
+        uint256 timestamp = addresses[vestedAddress].timestamp;
```

Comment:
maybe `timestamp` value need to be checked here for `!=0`, in fact even the `address(0)!=0` check makes more sense here than in other methods I think